docs/source/kubernetes/redhat/step-zero-openshift.md

```diff
@@ -4,13 +4,9 @@
 
 [OpenShift](https://www.okd.io/) from RedHat is a cluster manager based on Kubernetes.
 
-For setting up JupyterHub on OpenShift, check out the [JupyterHub on OpenShift](https://github.com/jupyter-on-openshift/jupyterhub-quickstart)
-project. It provides an OpenShift template based JupyterHub deployment. Zero to JupyterHub uses
-[helm](https://helm.sh) which is currently usable with OpenShift; yet deploying helm on OpenShift
-is somewhat complicated (see RedHat's blog post on [Getting Started with Helm on OpenShift](https://cloud.redhat.com/blog/getting-started-helm-openshift)).
+For running Z2JH on openshift, check out the [z2jh-openshift](https://github.com/gembaadvantage/z2jh-openshift) project. It customizes the containers used by the helm chart with security configuration required by OpenShift, and makes minor alterations to network policies to enable networking with the Weave NPC and the default OpenShift DNS.
 
-## Additional resources about Jupyter on OpenShift
+Otherwise for setting up alternative notebook environments, checkout:
```

Comment:
I'm happy to change it to the discourse discussion.

I initially listed other resources because it's what the OG RedHat engineer in this space, Graham Dumpleton, recommended when [he acknowledged](https://github.com/jupyter-on-openshift/jupyterhub-quickstart/issues/38#issuecomment-900629681) that [this repo](https://github.com/jupyter-on-openshift/jupyterhub-quickstart) isn't maintained anymore.